packages/cli/src/commands/__tests__/commands.spec.ts

```diff
@@ -0,0 +1,114 @@
+import * as utils from '@stoplight/prism-http';
```

Comment:
I do not know why I called it in this way. I'll rename.

I cannot use `jest.mock` because I need to mock the returned values in some functions, see line 10,11